internal/config/configdomain/proposal_stack_lineage_builder.go

```diff
@@ -0,0 +1,131 @@
+package configdomain
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/git-town/git-town/v21/internal/forge/forgedomain"
+	"github.com/git-town/git-town/v21/internal/git/gitdomain"
+	. "github.com/git-town/git-town/v21/pkg/prelude"
+)
+
+func NewProposalStackLineageBuilder(args *ProposalStackLineageArgs) Option[ProposalStackLineageBuilder] {
+	if args == nil || args.Connector.IsNone() {
```

Comment:
Rather than checking `IsNone` here, and then accessing this field using `GetOrPanic` on line 22, I suggest you properly unpack this optional value, return early if there is no connector, and then use the unpacked value on line 22.